views/random-beers.hbs

```diff
@@ -0,0 +1,10 @@
+{{#each beer }}
+    {{!-- <div><img class="beer-pic" src="{{ this.image_url }}"></div>
+    <div> {{ this.name }} </div>
+    <div> {{ this.description }} </div>
+    <div> {{ this.tagline }} </div> --}}
+    {{ > beerpartial this }}
```

Comment:
Don't add spaces before the > symbol when inserting partials. This may cause issues on some browsers.
`{{> beerpartial this}}`